### PR TITLE
fix(show-password): replace show password with icon

### LINKED ISSUE #6023: Replace Show password with 👁 icon
Our **Show** password button has a few problems that I think an icon will fix.

PROBLEMS
1. The label "Show" varies in length by language, but the visible area of the password itself does not. This leads to a bad situation where more of the the password is visible in some languages than others. A universal icon will equalize this.
1. "Show" is probably more understood as a way to toggle to visible and no simply peek (on press only)
1. In Internet Explorer an eye icon is used for the behaviour of "peeking" at the password. Likely more familiar to our large Windows user base.

SPECS
* Icon is centred in button space. Button space should match field height (e.g. 43 pixels)
* Visible password area can now be field width minus field height
* Button has a white background unless hovered or press. Eyeball is never shown on top of password.
* SVG icon: [password-show.svg.zip](https://github.com/mozilla/fxa-content-server/files/1880373/password-show.svg.zip)

METRICS
* Would be great to see if more users are making use of this than before, especially in other languages

eye password

child of #6083

### PR DESCRIPTION
**Pull request by divyabiyani**
Fixes #6023

### CODE REVIEW

**Comment by vladikoff** [comment]
Oh hey @divyabiyani  !

File: app/images/show-password.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (vladikoff) [inline app/images/show-password.svg]: could we try to run this svg through https://jakearchibald.github.io/svgomg/ ?

**Comment by vladikoff** [comment]
Let's do the changes made to `npm-shrinkwrap.json` :)

**Comment by vladikoff** [comment]
@divyabiyani I noticed this on sign up page:



the top eye gets fully covered with dark grey. I can try tweaking that on my own if you are too busy though 👍

**Comment by shane-tomlinson** [comment]
Hi @divyabiyani - You have done great work here! I am hoping to get this PR merged by Monday for the upcoming train cut. I know you are busy so don't want to add to your load. I am happy to finish this up, but don't want to step on your toes if you have a bit of time to finish it up.

**Comment by shane-tomlinson** [comment]
Replaced by #6184, closing.

(Sorry I stepped on your toes @divyabiyani, I need this functionality in by Monday!)

**Comment by divyabiyani** [comment]
Hey @vladikoff @shane-tomlinson, Sorry I got really stuck with some work and couldn't reply. 

And Thanks @shane-tomlinson for completing the work and sorry for the long communication gap.
Will be more active from next time.

**Pull request closed without merging**